superbench/benchmarks/micro_benchmarks/rocm_gemm_flops_performance.py

```diff
@@ -0,0 +1,162 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+"""Module of the FLOPs performance benchmarks."""
+
+import os
+
+from superbench.common.utils import logger
+from superbench.benchmarks import BenchmarkRegistry, Platform
+from superbench.benchmarks.micro_benchmarks import GemmFlopsBenchmark
+
+
+class RocmGemmFlopsBenchmark(GemmFlopsBenchmark):
+    """The GEMM FLOPs performance benchmark class."""
+    def __init__(self, name, parameters=''):
+        """Constructor.
+
+        Args:
+            name (str): benchmark name.
+            parameters (str): benchmark parameters.
+        """
+        super().__init__(name, parameters)
+
+        self._bin_name = 'rocblas-bench'
+        self._support_precisions = ['FP64', 'TF32_TC', 'FP16_TC', 'BF16_TC', 'INT8_TC']
+        self.__precision_and_kernel_map = {
+            'FP64': '-r f64_r -f gemm',
+            'TF32_TC': '-r f32_r -f gemm_ex --compute_type f32_r',
+            'FP16_TC': '-r f16_r -f gemm_ex --compute_type f32_r',
+            'BF16_TC': '-r bf16_r -f gemm_ex --compute_type f32_r',
+            'INT8_TC': '--a_type i8_r --b_type i8_r --c_type i32_r --d_type i32_r -f gemm_ex --compute_type i32_r'
```

Comment:
TC is incorrect because it's nvidia term. We should change it to xDLOPS

Reply:
renamed, thanks